<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BNE3700" failures="2" tests="12" name="Test" time="146.984" errors="0" timestamp="2024-12-30T16:36:46 IST">
  <testcase classname="testCases.TC101_SignUpTest" name="testSignUpRedirection" time="4.097"/>
  <testcase classname="testCases.TC102_LoginTest" name="testValidLoginPage" time="7.113"/>
  <testcase classname="testCases.TC102_LoginTest" name="testInvalidLogin" time="3.276"/>
  <testcase classname="testCases.TC103_ContactAdditionTest" name="testContactListMandatoryfileds" time="11.626"/>
  <testcase classname="testCases.TC103_ContactAdditionTest" name="testContactListoptionalfileds" time="6.767"/>
  <testcase classname="testCases.TC103_ContactAdditionTest" name="duplicateContactList" time="11.14">
    <failure type="java.lang.AssertionError" message="Expected error message for duplicate contact, but got:  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected error message for duplicate contact, but got:  expected [true] but found [false]
at testCases.TC103_ContactAdditionTest.duplicateContactList(TC103_ContactAdditionTest.java:97)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- duplicateContactList -->
  <testcase classname="testCases.TC104_ContactsDisplayTest" name="testContactsAlphabeticalOrder" time="13.589"/>
  <testcase classname="testCases.TC104_ContactsDisplayTest" name="testPhonenumberwithExtension" time="3.058">
    <failure type="java.lang.AssertionError" message="phone numbers doesn&amp;apos;t have extensions expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: phone numbers doesn't have extensions expected [true] but found [false]
at testCases.TC104_ContactsDisplayTest.testPhonenumberwithExtension(TC104_ContactsDisplayTest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testPhonenumberwithExtension -->
  <testcase classname="testCases.TC105_ContactEditingTest" name="testContactsEditing" time="19.677"/>
  <testcase classname="testCases.TC105_ContactEditingTest" name="TestEmptyfields" time="9.725"/>
  <testcase classname="testCases.TC106_ContactDeleteTest" name="testContactDelete" time="12.956"/>
  <testcase classname="testCases.TC107_LogoutTest" name="testLogout" time="6.863"/>
</testsuite> <!-- Test -->
